tests/Specs.hs

```diff
@@ -35,4 +38,31 @@ runSpecs = do
     iterSolution_spec
     iterSolutionWithKey >>= print
 
+  putStrLn "Checking entangleShape"
```

Comment:
I would put these tests in a separate file; the `Specs` file is for unit testing particular strictness specifications.